queen-attack/example.nim

```diff
@@ -1,18 +1,12 @@
-type
-  Position* = tuple[rank, file: int]
-
-proc isValid(white, black: Position): bool =
+proc isValid(white, black: tuple[rank, file: int]): bool =
```

Comment:
I think philosophically if we can provide a test suite that will let people solve the problem in different ways, that's usually preferable to something that forces a specific implementation. I'm not sure that that applies here, though.